pymlx/model.py

```diff
@@ -54,20 +54,32 @@ def bulk_predict(self, test_data):
     #   - Absorb xgboost
     #   - Each predictor should indicate whether it supports features importance
     #   11/09/2018: has just implement for xgboost and lightgbm
-    def get_fscores(self):
+    def get_fscores(self, no_features_name=True):
         if self.predictor.__module__ == 'xgboost.sklearn':
             # for xgboost
             # feature_names = self.featurizer.out_feature_names
-            fscores = self.predictor.get_booster().get_fscore()
+            _fscores = self.predictor.get_booster().get_fscore()
+            fscores = {}
+            if not no_features_name: # when the training data is dataframe
+                fscores = _fscores
+            else:
+                for k, v in _fscores.items():
+                    fscores[self.featurizer.out_feature_names[int(k.replace('f', ''))]] = v
+
             # fscores = {feature_names[int(f[1:])]: fscores[f] for f in fscores}
             return Series(fscores).sort_values(ascending=False)
         else:
             # for lightgbm
             fscores = {}
             booster = self.predictor.booster_
             fimportances = booster.feature_importance()
+            # print('light gbm', fimportances)
             for i, fname in enumerate(booster.feature_name()):
-                fscores[fname] = fimportances[i]
+                if not no_features_name:
```

Comment:
What does `no_feature_name` mean? 

Whatever it is, `if not no_feature_name` sounds hard to read. Better change to `if no_feature_name ... else ...`.

Reply:
Đã cập nhật.

Khi dữ liệu train và test là numpy array thì lúc lấy feature importance của model nó sẽ cho ra tên các feature dạng f1, f2, f3,... đối với xgboost, hoặc Column_1, Column_2, Column_3,... đối với lightgbm.

`no_feature_name`=True khi cần thay thế f1, f2, f3,... hoặc Column_1, Column_2, Column_3,... thành tên feature cho dễ đọc.